SOSS/extract/Simple_solver/simple_solver.py

```diff
@@ -17,11 +17,12 @@
 import warnings
 import emcee
 from scipy.ndimage.interpolation import rotate
-from SOSS.trace import get_uncontam_centroids as ctd
+from SOSS.dms import soss_centroids as ctd
```

Comment:
SOSS should be lower case
```python
from soss.dms import soss_centroids
```
avoid shortening too many imports as people without pycharm may find it hard to read

Reply:
Address during review of simple solver.